src/widgets/newsFeed/NewsFeed.tsx

```diff
@@ -0,0 +1,37 @@
+import { useQuery } from "react-query";
+
+import styles from './news-feed.module.css'
+import Card from "~entities/card";
+import { Button } from "~shared/ui/button";
+import { Spinner } from "~shared/ui/spinner";
+import { ArticleDTO } from "~shared/api/Api.ts";
+
+// import { useInfiniteArticles } from "~widgets/newsFeed/api/feedApi.tsx";
+
+export function NewsFeed() {
+  const { isLoading, data: newsFeed, error } = useQuery({
+    queryKey: [ 'newsFeed' ],
+    queryFn: async () => {
```

Comment:
looks like it should be in entities/article/model

Reply:
It will be. It just a draft